packages/fbjs/package.json

```diff
@@ -63,7 +63,6 @@
     ]
   },
   "dependencies": {
-    "cross-fetch": "^3.1.5",
```

Comment:
Side-note, I am not sure why this was included as a dependency, it seems like this was only ever used during testing.

Reply:
fbjs was created to support FB projects, goal being to provide the same core/shared libraries we had internally. `fetch` was one of those things but had too much internal-specific code so it was a "forked" implementation here. It relied on `cross-fetch` and re-exported it (as you saw) but was not otherwise used directly.